### PR TITLE
make splitview not culture sensitive.

### PR DESCRIPTION
**Pull request by danwalmsley**
SplitView could have layout issues on some cultures.

### CODE REVIEW

File: src/Avalonia.Controls/SplitView.cs [hunk 1]
```diff
@@ -431,18 +431,40 @@ private void OnCompactPaneLengthChanged(AvaloniaPropertyChangedEventArgs e)
             }
         }
 
+        private string GetPsuedoClass(SplitViewDisplayMode mode)
```
Comment (jp2masa) [inline src/Avalonia.Controls/SplitView.cs:434]: `GetPsuedoClass` -> `GetPseudoClass`

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021669-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0021677-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: src/Avalonia.Controls/SplitView.cs (hunk 1 above)
Comment (danwalmsley) [reply src/Avalonia.Controls/SplitView.cs]: fixed

**Review by maxkatz6 — APPROVED** [review]

**Pull request merged**